search/tests/test_indexing.py

```diff
@@ -101,10 +110,15 @@ def test_sorting(self):
         )
         # expected position (res2, res3, res1)
         top_res = results[0]
+        sec_res = results[1]
         self.assertEqual(
             top_res.nr_views,
             res2.xa_nr_views
         )
+        self.assertEqual(
```

Comment:
I think the test of the proper order is implicit:
1- res 4 is created after the others and has all the top values of the sorting
2- since res4 has the top value of each sorting, but a higher ID, it appears always as second in the sorting